package.json

```diff
@@ -789,6 +789,7 @@
     "yargs": "^16.2.0"
   },
   "dependencies": {
+    "@types/node-fetch": "^2.6.2",
```

Comment:
keep the version same as in dependencies

Reply:
npm ERR! notarget No matching version found for @types/node-fetch@^2.6.7.